Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -198,6 +198,9 @@ protected virtual void OnElementChanged(ElementChangedEventArgs<Label> e)
 
 		protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
+			if (_disposed)
```

Comment:
On the right track with this today. Manually firing off GC collection is starting to break things. Putting together a repro project this afternoon,